### PR TITLE
function to change width of bars in barchart

### PR DESCRIPTION
**Pull request by ayisharuna**

### CODE REVIEW

File: app/assets/scripts/modules/setBarWidth.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+* test module for changing the width of bars
+*/
+const ChartFactory = require('../../../helpers/chartFactory');
+const _  = require('lodash');
+
+const setBarWidth = (chart, params) => {
+  _.each(chart.data, (d) => {
+    d.width = params.newBarWidth;
```
Comment (AhmetTanakol) [inline app/assets/scripts/modules/setBarWidth.js]: instead of changing all the data, we can specify which trace will be changed

**Comment by AhmetTanakol** [comment]
Please resolve conflicts, fix lint and test errors

**Comment by hubjak** [comment]
Fixed the test errors.

**Review by hubjak — APPROVED** [review]

File: test/bar.test.js [hunk 2]
```diff
@@ -82,8 +87,10 @@ describe('BarChart Tests', () => {
       marker: {
         color: 'red',
       },
+      width: 0.3,
       name: 'Trace 1',
     }];
+    console.log(testBarChart.layout);
```
Comment (AhmetTanakol) [inline test/bar.test.js:97]: you can remove `console.log`

File: app/assets/scripts/modules/changeBarWidth.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+* test module for changing the width of bars
+*/
+const ChartFactory = require('../../../helpers/chartFactory');
+const _  = require('lodash');
+
+const setBarWidth = (chart, params) => {
+ return chart;
+  _.each(chart.data, (d) => {
+    d.width = params.newBarWidth;
+    console.log(params.newBarWidth);
```
Comment (AhmetTanakol) [inline app/assets/scripts/modules/changeBarWidth.js:11]: you can remove `console.log`

**Review by AhmetTanakol — COMMENTED** [review]
Missing changing color of bars
Conflicts should be resolved

**Pull request merged**